src/components/MultiHopTrade/components/RateGasRow.tsx

```diff
@@ -23,6 +24,7 @@ type RateGasRowProps = {
   swapperName: SwapperName | undefined
   swapSource: SwapSource | undefined
   networkFeeFiatUserCurrency: string | undefined
+  deltaPercentage?: BigNumber | null
```

Comment:
Nitpick: pass me as string which is always the better, more serializable option

Reply:
Changed in [e905124](https://github.com/shapeshift/web/pull/8970/commits/e9051249766cda9ac47762f1ce453ba3f1e35c9f)